docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+- "cluster-config-v1" ConfigMap since versions 4.9+
 
-* "cluster-monitoring-config" ConfigMap data since versions:
-  - 4.6.22+
-  - 4.7+
+### Anonymization
+If the content of a ConfigMap contains a parseable PEM structure (like a certificate), it removes the inside of
+PEM blocks. For ConfigMap of type BinaryData, it is encoded as standard base64. In the archive under configmaps,
+we store the name of the namespace, the name of the ConfigMap, and then each ConfigMap Key.
 
-* "cluster-config-v1" ConfigMap since versions:
-  - 4.9+
+For example: ```config/configmaps/NAMESPACENAME/CONFIGMAPNAME/CONFIGMAPKEY1```
 
 
 ## ContainerImages
 
-collects essential information about running containers.
-Specifically, the age of pods, the set of running images and the container names are collected.
+Collects essential information about running containers. Specifically, the age of pods,
+the set of running images and the container names are collected.
 
-* Location in archive: config/running_containers.json
-* Id in config: clusterconfig/container_images
-* Since versions:
-  - 4.5.33+
-  - 4.6.16+
-  - 4.7+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/config/running_containers.json](./insights-archive-sample/config/running_containers.json)
+
+### Location in archive
+- `config/running_containers.json`
+
+### Config ID
+`clusterconfig/container_images`
+
+### Released version
+- 4.7.0
+
+### Backported versions
+- 4.5.33+
+- 4.6.1+
+
+### Changes
+None
 
 
 ## ContainerRuntimeConfig
 
-collects ContainerRuntimeConfig  information
+Collects ContainerRuntimeConfig information.
+
+### API Reference
+- https://github.com/openshift/machine-config-operator/blob/master/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L402
+- https://docs.okd.io/latest/rest_api/machine_apis/containerruntimeconfig-machineconfiguration-openshift-io-v1.html
 
-The Kubernetes api:
+### Sample data
+- [docs/insights-archive-sample/config/containerruntimeconfigs/set-log-and-pid.json](./insights-archive-sample/config/containerruntimeconfigs/set-log-and-pid.json)
 
-	https://github.com/openshift/machine-config-operator/blob/master/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L402
+### Location in archive
+- `config/containerruntimeconfigs/{name}.json`
 
-Response see:
+### Config ID
+`clusterconfig/container_runtime_configs`
 
-	https://docs.okd.io/latest/rest_api/machine_apis/containerruntimeconfig-machineconfiguration-openshift-io-v1.html
+### Released version
+- 4.7.0
 
-* Location in archive: config/containerruntimeconfigs/
-* Id in config: clusterconfig/container_runtime_configs
-* Since versions:
-  - 4.6.18+
-  - 4.7+
+### Backported versions
+- 4.6.18+
+
+### Changes
+None
 
 
 ## ContainersLogs
 
-collects either current or previous containers logs for pods firing one of the configured alerts.
+Collects either current or previous containers logs for pods firing one of the
+configured alerts.
```

Comment:
Are the "configured alerts" hardcoded or this is a conditional gatherer?

Reply:
Alerts use to be in our code, but now they are fetched from the service. I'll update it.